vendor/google-maps.js

```diff
@@ -0,0 +1,25 @@
+
+
+window.google = window.google || {};
+google.maps = google.maps || {};
```

Comment:
Okay, so my guess is that `google.maps` on the right is invalid due to being undefined, which is why we get the empty undefined object on the furthest right. I think. Regardless, `google.maps` needs to get defined somehow, which should be defined by the function below, but that isn't happening for some reason.